pandas/core/algorithms.py

```diff
@@ -1305,6 +1305,8 @@ def compute(self, method: str) -> Series:
         narr = len(arr)
         n = min(n, narr)
 
+        # arr passed into kth_smallest must be contiguous, which
```

Comment:
use ascontiguousarray instead (or specify the ordering flag in copy)

Reply:
done